pkg/vgmanager/lvm.go

```diff
@@ -124,11 +137,55 @@ func ListVolumeGroups() ([]*VolumeGroup, error) {
 	}
 	groups := []*VolumeGroup{}
 	for _, info := range infoList {
-		groups = append(groups, &VolumeGroup{info["vg_name"]})
+		// get all PVs which are part of VG
+		pvList, err := parseOutput("pvs", "pv_name", "-S", fmt.Sprintf("'vg_name=%s'", info[VGName]))
+		if err != nil {
+			return nil, err
+		}
+		pvs := make([]string, 0, len(pvList))
+		for _, pv := range pvList {
+			pvs = append(pvs, pv[PVName])
+		}
+		groups = append(groups, &VolumeGroup{
+			name: info[VGName],
+			pvs:  pvs,
+		})
 	}
 	return groups, nil
 }
 
+// RemoveVolumeGroup removes volume group and physical volumes that are part of it
+func RemoveVolumeGroup(name string) error {
+
+	// find volume group which contains name and list of PVs belonging to VG
+	vg, err := FindVolumeGroup(name)
```

Comment:
Unrelated to this pr but don't export this function if it is not being used outside the pkg.

Reply:
- there are many functions like this, will take this in a separate PR